src/content/docs/en/guides/images.mdx

```diff
@@ -159,6 +159,37 @@ import myImage from '../assets/my_image.png'; // Image is 1600x900
 
 <ReadMore>See details about [the `<Picture />` component properties](/en/reference/modules/astro-assets/#picture-properties) in the `astro:assets` reference.</ReadMore>
 
+### Inline `.svg` files in the HTML
```

Comment:
```suggestion
### Inline `.svg` files in the HTML (experimental)
```
Looking at the source PR, it seems this will be behind an experimental flag?

CURRENTLY, **everything** about experimental features is only documented in config reference docs, but I know everyone hates that. So for v5 we can instead do something like this, but then we'll also need to link back to experimental config reference docs somehow to tell people how they can enable it.

OR **all** of this documentation exists only within the JSDOC of that PR.